### PR TITLE
.vscode: add basic go settings

### PR DESCRIPTION
**Pull request by zdylag**
Test coverage is generally useful, so enable it for most test runs.

Listing the tests run and enabling race-detection is also useful
for identifying issues.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "go.coverOnSingleTest": true,
+    "go.coverOnSingleTestFile": true,
+    "go.testFlags": ["-v", "-race"]
+}
```
Comment (Matt-Hurd) [inline .vscode/settings.json:5]:
```suggestion
}

```

Comment (zdylag) [reply .vscode/settings.json]: I may or may not have added this. It seems different on my current machine (Windows is new to me) and the github view.

**Comment by JoshSanch** [comment]
It might be better to document these rather than explicitly override settings.json for each developer.

**Comment by zdylag** [comment]
@JoshSanch I am also open to that! Do you have ideas for a good way to document settings like this so that new developers in the repo are likely to use them?

These _do_ only override vscode settings in case people happen to be using that. So maybe that also helps either make this okay, or give us a reason this is not okay.

**Comment by zdylag** [comment]
Closing this in favor of some documentation 👍

**Pull request closed without merging**